tests/bearlib/aspects/ClassTest.py

```diff
@@ -31,3 +33,10 @@ class docs:
                 example = 'Example'
 
         assert not SubAspect.docs.check_consistency()
+        assert issubaspect(SubAspect, RootAspect)
+        assert not issubaspect(Root, RootAspect)
+        assert not issubaspect(RootAspect, RootAspect)
+        with pytest.raises(TypeError):
+            issubaspect('String', SubAspect)
+        with pytest.raises(TypeError):
+            issubaspect(RootAspect, str)
```

Comment:
As @Techievena mentioned in https://github.com/coala/coala/pull/3996#discussion_r108064910 all the `issubaspect` tests should really be put in a separate function outside of `AspectClassTest`, maybe even in a separate test module.